components/kyma-environment-broker/internal/cls/client.go

```diff
@@ -87,3 +102,31 @@ func createParameters(config *Config) parameters {
 	params.SAML.Sp.SignaturePrivateKey = config.SAML.Sp.SignaturePrivateKey
 	return params
 }
+
+type bindParam struct{}
+
+func (c *Client) CreateBinding(smClient servicemanager.Client, request *BindingRequest) (*ClsOverrides, error) {
+	var bp bindParam
+
+	respBinding, err := smClient.Bind(request.InstanceKey, request.BindingID, bp, false)
+	if err != nil {
+		return nil, errors.Wrapf(err, "Bind() call failed")
+	}
+	// get overrides
+	clsOverrides, err := getCredentials(respBinding.Binding)
+	if err != nil {
+		return nil, errors.Wrapf(err, "getCredentials() call failed")
+	}
+	return clsOverrides, nil
+
+}
+
+func getCredentials(binding servicemanager.Binding) (*ClsOverrides, error) {
+	clsOverrides := ClsOverrides{}
+	credentials := binding.Credentials
```

Comment:
We can initialize the struct like this instead. It would be simpler:
```suggestion
	credentials := binding.Credentials{
	  KibanaUrl : credentials["Kibana-endpoint"].(string)
	}
```